packages/atomic/src/components/search/facets/atomic-automatic-facet-generator/atomic-automatic-facet-generator.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import defaultStory from 'atomic-storybook/default-story';
+import {html} from 'lit-html';
+
+const {defaultModuleExport, exportedStory} = defaultStory(
+  'Atomic/AutomaticFacetGenerator',
+  'atomic-automatic-facet-generator',
+  {desiredCount: '2', areCollapsed: 'false'},
+  {
```

Comment:
We could try to configure something (either in the admin/query pipeline, or adding a query that returns "proper" facet) so as to not get random facet with GUID being shown in the example.